### PR TITLE
TRAN-170: Added user authentication to builtin application

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://sd-transit.atlassian.net/browse/TRAN-170

### CODE REVIEW

File: transit/rest_api/superset/superset_auth.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+from rest_framework.decorators import (
+    api_view,
+    authentication_classes,
+    permission_classes
+)
+from rest_framework.response import Response
+from rest_framework_simplejwt import authentication
+
+
+SUPERSET_HOST = os.getenv('SUPERSET_HOST', 'http://localhost:8088/')
```
Comment (dborowiecki) [inline transit/rest_api/superset/superset_auth.py]: I think we should move this env variables to settings.py

**Review by dborowiecki — CHANGES_REQUESTED** [review]

File: transit/rest_api/superset/superset_auth.py (hunk 1 above)
Comment (sniedzielski) [reply transit/rest_api/superset/superset_auth.py]: ok, Good point. I will move those env into setting.py.

**Comment by sniedzielski** [comment]
@dborowiecki I've just fixed point you mentioned  to improve. Please re-check.

File: transit/rest_api/superset/superset_auth.py (hunk 1 above)
Comment (sniedzielski) [reply transit/rest_api/superset/superset_auth.py]: fixed

**Review by dborowiecki — APPROVED** [review]

**Pull request merged**